apache/Dockerfile

```diff
@@ -108,6 +108,14 @@ ENV DOCKER_MAUTIC_WORKERS_CONSUME_EMAIL=2 \
 
 COPY ./common/supervisord.conf /etc/supervisor/conf.d/supervisord.conf
 
+# Install composer
+RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/bin --filename=composer
```

Comment:
we don't need to redownload and setup here we can simply `cp --from=builder` because we already downloaded and setup [here](https://github.com/mautic/docker-mautic/pull/361/files#diff-02c190dd34ee6ce466ae02364d954e15a049a0343ba71774688653550e4c82f6R52)